### PR TITLE
Sb 24484 pk

### PR DESCRIPTION
**Pull request by pradeepkumarcm96**

### CODE REVIEW

File: src/app/client/src/app/modules/questionnaire/components/input-type-attachment/input-type-attachment.component.ts [hunk 1]
```diff
@@ -66,4 +71,37 @@ export class InputTypeAttachmentComponent implements OnInit {
...
+
+  async onAddApproval(file){
+    let metaData=this.observationUtil.getAlertMetaData();
+    metaData.type="upload evidence";
+    metaData.size="tiny";
+    metaData.content.title=this.resourceService.frmelmnts.alert.uploadevidencetitle;
+    metaData.content.body.type="checkbox";
+    let html =
+    `
+    ${this.resourceService.frmelmnts.alert.evidence_content_policy}<a href='https://diksha.gov.in/term-of-use.html' target="_blank">${this.resourceService.frmelmnts.alert.evidence_content_policy_label}</a> .${this.resourceService.frmelmnts.alert.uploadevidencecontent}
```
Comment (kiranharidas187) [inline src/app/client/src/app/modules/questionnaire/components/input-type-attachment/input-type-attachment.component.ts:83]: dont hardcode the url like this.. please check in the portal how teams and condition URLs are fetched

File: src/app/client/src/app/modules/shared/components/alert-modal/alert-modal.component.scss [hunk 2]
```diff
@@ -29,13 +29,14 @@
 }
 .popup-body-checkbox {
   position: relative;
-  top: 0px;
-  left: 20px;
-  font-size: 15px;
+  top: -18px;
```
Comment (kiranharidas187) [inline src/app/client/src/app/modules/shared/components/alert-modal/alert-modal.component.scss:32]: should not use negative px

File: src/app/client/src/app/modules/core/components/content-type/content-type.component.ts [hunk 3]
```diff
@@ -168,11 +165,7 @@ export class ContentTypeComponent implements OnInit, OnDestroy {
     };
     this.formService
       .getFormConfig(formServiceInputParams)
-      .subscribe((data: any) => {
-        this.utilService.formData = data;
```
Comment (kiranharidas187) [inline src/app/client/src/app/modules/core/components/content-type/content-type.component.ts:172]: why is this removed

Comment (pradeepkumarcm96) [reply src/app/client/src/app/modules/core/components/content-type/content-type.component.ts:172]: this.utilService.formData = data; is not need i just used for checking purpose only so i removed unwanted code

File: src/app/client/src/app/modules/shared/components/alert-modal/alert-modal.component.scss (hunk 2 above)
Comment (pradeepkumarcm96) [reply src/app/client/src/app/modules/shared/components/alert-modal/alert-modal.component.scss:32]: i will remove it
